specification/confidentialledger/Microsoft.CodeTransparency/main.tsp

```diff
@@ -28,10 +27,793 @@ namespace Microsoft.CodeTransparency;
 
 @doc("The Microsoft.CodeTransparency service versions.")
 enum Versions {
+  #deprecated "Use next version"
   @useDependency(Azure.Core.Versions.v1_0_Preview_1)
   @doc("The 2024-01-11-preview version of the Microsoft.CodeTransparency service.")
   `2024-01-11-preview`,
+
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  @doc("The 2025-01-31-preview version of the Microsoft.CodeTransparency service.")
+  `2025-01-31-preview`,
 }
 
 // Additional supported traits by the API.
 alias ServiceTraits = SupportsClientRequestId;
+
+// -----------------------------------------------------------------
+// Service configuration endpoints
+// -----------------------------------------------------------------
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@added(Versions.`2025-01-31-preview`)
+@doc("Get the transparency configuration file, see IETF SCITT SCRAPI draft")
+@sharedRoute
+@route("/.well-known/transparency-configuration")
+@get
+op getTransparencyConfigCbor is Foundations.Operation<
+  {
+    @doc("Accpect header")
+    @header
+    accept: "application/cbor";
+  },
+  TransparencyConfigurationCbor,
+  ServiceTraits,
+  CborServerErrors
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@added(Versions.`2025-01-31-preview`)
+@doc("Get the transparency configuration file, see IETF SCITT SCRAPI draft")
+@sharedRoute
+@route("/.well-known/transparency-configuration")
+@get
+op getTransparencyConfigJson is Foundations.Operation<
+  {
+    @doc("Accpect header")
+    @header
+    accept: "application/json";
+  },
+  TransparencyConfigurationJson,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@added(Versions.`2025-01-31-preview`)
+@doc("Get the service public keys")
+@route("/jwks")
+@get
+op getPublicKeys is Foundations.Operation<
+  {},
+  JwksDocument,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get service parameters")
+@route("/parameters")
+@get
+op getParameters is Foundations.Operation<{}, ParametersResult, ServiceTraits>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@removed(Versions.`2025-01-31-preview`)
+@doc("Get the DID configuration file.")
+@route("/.well-known/did.json")
+@get
+op getDidConfig is Foundations.Operation<{}, DidDocument, ServiceTraits>;
```

Comment:
Do we have a way to/want to mark those as deprecated, and direct usage towards the transparency-configuration endpoint instead?